csvcubed/tests/unit/utils/qb/validation/test_observations.py

```diff
@@ -57,5 +62,59 @@ def test_find_sdmxa_obs_status_columns():
     }, obs_status_col_names
 
 
+def test_value_uri_template_is_present_in_existing_measure_dimention():
+    """
+    Testing to see that when the measure dimention is defined, that the value uri template is also defined.
+    """
+    qube = Cube(
+        metadata=CatalogMetadata("Some Qube"),
+        data=None,
+        columns=[
+            QbColumn("Some Dimension", NewQbDimension(label="Some Dimension")),
+            QbColumn(
+                "Values",
+                QbMultiMeasureObservationValue(
+                    unit=NewQbUnit("Some Unit"),
+                ),
+            ),
+            QbColumn(
+                "Measure", 
+                QbMultiMeasureDimension([
+                    ExistingQbMeasure("http://some-measure")
+                ]),
+                csv_column_uri_template="http://some-uri/{+measure}"
+            )
+        ],
+    )
+    errors = _validate_multi_measure_cube(qube, None)
+    assert len(errors) == 0, [e.message for e in errors]
+
+
+def test_value_uri_template_is_missing_in_existing_measure_dimention():
+    """
+    Testing to see that the value uri template is not defined when the measure dimention is defined.
+    """
+    qube = Cube(
+        metadata=CatalogMetadata("Some Qube"),
+        data=None,
+        columns=[
+            QbColumn("Some Dimension", NewQbDimension(label="Some Dimension")),
+            QbColumn(
+                "Values",
+                QbMultiMeasureObservationValue(
+                    unit=NewQbUnit("Some Unit"),
+                ),
+            ),
+            QbColumn(
+                "Measure", 
+                QbMultiMeasureDimension([
+                    ExistingQbMeasure("http://some-measure")
+                ]),
+            )
+        ],
+    )
+    errors = _validate_multi_measure_cube(qube, None)
+    assert len(errors) == 1, [e.message for e in errors]
```

Comment:
Can you add some assertions to ensure the error is of the expected type and points at the right type of component?